code/__HELPERS/game.dm

```diff
@@ -395,6 +395,48 @@
 
 	return new /datum/projectile_data(src_x, src_y, time, distance, power_x, power_y, dest_x, dest_y)
 
+/proc/pollCultists()
+	var/time_passed = world.time
+	set waitfor = FALSE
+	var/list/yes_voters = new
+	var/list/cult_total = new
+	for(var/mob/M in mob_list)
+		spawn(0)

```

Comment:
We don't allow this.
You've also not indented the code underneath it, so it's also doing nothing.